setup.cfg

```diff
@@ -41,5 +44,12 @@ extend-ignore =
 exclude =
     build/
     venv/
-    traveltimepy/TimeFilterFastRequest_pb2.py
-    traveltimepy/TimeFilterFastResponse_pb2.py
\ No newline at end of file
+    traveltimepy/proto/
+
+[mypy]
+ignore_missing_imports = True
+exclude = ^(traveltimepy/proto/.*|build/.*|venv/.*)$
+
+[mypy-traveltimepy/proto.*]
+follow_imports = skip
```

Comment:
locally it works fine without this for me, but in CI it still checks the proto files if this is not here